packages/configuration-backend/src/service.ts

```diff
@@ -233,6 +266,62 @@ export const getMerchantModels = async (
   }
 }
 
+function getAllowedKeysToMerchant() {
+  return [
+    'merchantName',
+    'merchantStreet',
+    'merchantZip',
+    'merchantCity',
+    'merchantEmail',
+    'merchantPhone',
```

Comment:
Here should be merchantShopId also. I think it was under merchantPhone earlier (maybe the order matters if it is checked in some unit test?)

Reply:
This fetches only the service configuration, which should not countain that merchantShopId because it is only available from merchant api

Reply:
But yeh, i should add that because this is used in both functions